tests/test_eload_ingestion.py

```diff
@@ -206,6 +212,28 @@ def test_insert_browsable_files(self):
             self.eload.insert_browsable_files()
             m_execute.assert_not_called()
 
+    def test_update_browsable_files_with_date(self):
+
+        with patch('eva_submission.eload_submission.get_metadata_connection_handle', autospec=True), \
+                patch('eva_submission.eload_ingestion.execute_query') as m_execute:
+            self.eload.update_browsable_files_with_date()
+            m_execute.assert_called()
+
+    def test_update_files_with_ftp_path(self):
+        with patch('eva_submission.eload_submission.get_metadata_connection_handle', autospec=True), \
+                patch('eva_submission.eload_ingestion.get_all_results_for_query') as m_get_results, \
+                patch('eva_submission.eload_ingestion.execute_query') as m_execute:
+            m_get_results.side_effect = [[(1, 'filename_1')], []]
+            self.eload.update_files_with_ftp_path()
+            m_execute.assert_called()
+
+            # calling insert again fail because no files are present
+            m_execute.call_count = 0
+            with pytest.raises(ValueError):
```

Comment:
I think elsewhere we're using `self.assertRaises` (should also make the pytest import unnecessary)